### DISCUSSION TITLE
Allow Frozen Columns

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6693] [Serenity Ideas]
The newest version of Serenity no longer support Slickgrid Frozen columns. It would be great if this feature was brought back.

**Comment by vooba** [comment]
Frozen Columns are working with v6.5.1

```Typescript
protected getSlickOptions() {
        let opt = super.getSlickOptions();
        opt.layoutEngine = new FrozenLayout();
        opt.frozenColumns = 1;
        return opt;
    } 
```

**Comment by ibrahimGit123** [comment]
I do see that, but doing this causes incompatibility issues when combined with Totals (CustomSummaryMixin), where the totals don't work properly
